cdap-docs/hydrator-manual/source/_includes/plugins/analytics/index.rst

```diff
@@ -0,0 +1,34 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2016 Cask Data, Inc.
+
+.. _cask-hydrator-analytic-plugins:
+
+================
+Analytic Plugins
+================
+
+Hydrator Version |cask-hydrator-version|
+
+These plugins work with:
+
+  - *Data Pipeline - Batch* (``cdap-data-pipeline`` artifact)
+  - *Data Pipeline - Realtime* (``cdap-data-streams`` artifact)
+
+.. toctree::
+    :maxdepth: 1
+    :glob:
+
+    [!D][!P][!B]*
```

Comment:
Globbing is limited, and doesn't support patterns as you suggest. Fixed the plugin prefixed for the `DPB` (Data Pipeline Batch).